how-to/create-working-group.md

```diff
@@ -0,0 +1,9 @@
+This document describes some part of how to create a working group in the InnerSource Commons.
```

Comment:
```suggestion
# Creating a Working Group in the InnerSource Commons

This document describes some part of how to create a working group in the InnerSource Commons.
```

Does anything speak against adding a title to this doc?

Also is the "some part of" significant, or could it be dropped?